language/stdlib/modules/Roles.move

```diff
@@ -504,40 +513,52 @@ module Roles {
     /// TODO(wrwg): link to requirements
 
     spec module {
-        /// Validator roles are only granted by LibraRoot [B4]. A new `RoleId` with `VALIDATOR_ROLE_ID()` is only
+        /// the LibraRoot role is only granted in genesis [B2]. A new `RoleId` with `LIBRA_ROOT_ROLE_ID` is only
```

Comment:
s/the/The. Here and elsewhere, please use full English sentences.

Reply:
Done.

Reply:
Done.